<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listgui</class>
 <widget class="QDialog" name="listgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="Relations">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Scientists</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="input_filter_scientists">
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sort by:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortSciByName">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortSciBySex">
           <property name="text">
            <string>Sex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortSciByBirth">
           <property name="text">
            <string>Birth Year</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortSciByDeath">
           <property name="text">
            <string>Death Year</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_scientists">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sex</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Born</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Death</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="button_add_scientist">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_remove_scientist">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_edit_scientist">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Computers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="input_filter_computers">
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sort by:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortCompByName">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortCompByBuildYear">
           <property name="text">
            <string>Build year</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortCompByType">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortCompByWasBuilt">
           <property name="text">
            <string>Was built</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_computers">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>wasBuilt</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="button_add_computer">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_remove_computer">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_edit_computer">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Relations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Sort by:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortBySci">
           <property name="text">
            <string>Scientist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbutton_sortByComp">
           <property name="text">
            <string>Computer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_relations">
         <column>
          <property name="text">
           <string>Scientist Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Computer Name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="button_add_relation">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_remove_relation">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_edit_relation">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
